src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -391,6 +391,13 @@ export class VisualizeEmbeddable
       syncColors: this.input.syncColors,
       uiState: this.vis.uiState,
       inspectorAdapters: this.inspectorAdapters,
+      executionContext: this.deps.start().core.executionContext.create({
+        type: this.vis.params.type,
+        name: this.vis.type.name,
+        id: this.vis.id ?? 'unknown_id',
+        description: this.vis.title ?? this.vis.type.title,
```

Comment:
Maybe the title should be used as `name` and `this.vis.description` should be used as the description?

Reply:
Name is a public name of `type`, because sometimes we report such discrepancy: `"type":"gauge","name":"tagcloud"`

Reply:
`name` is documented as the `public name of a user-facing feature` so I think providing the name of the _type_ of visualization better maps to that field. `description` is better suited for the specific _instance_ of the feature, like the specific visualization title.